sql/mysqld.cc

```diff
@@ -454,6 +454,7 @@ uint opt_binlog_gtid_index_span_min= 65536;
 my_bool opt_master_verify_checksum= 0;
 my_bool opt_slave_sql_verify_checksum= 1;
 const char *binlog_format_names[]= {"MIXED", "STATEMENT", "ROW", NullS};
+ulong binlog_error_action;
```

Comment:
A passerby colleague questioned why a variable for a single file requires modifying 4 files to add.

I’ll move this like to `log.cc` and then, yes, why is a variable split among variables`.cc`, implementation`.cc` and a bridging `.h` 🤷?